plugins/org.jkiss.dbeaver.ext.intersystems/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jkiss.dbeaver</groupId>
+    <artifactId>plugins</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+     <relativePath>../</relativePath>
+  </parent>
+  <artifactId>org.jkiss.dbeaver.ext.intersystems</artifactId>
```

Comment:
Your Pom file is too big and has too many dependencies. You can copy the pom file from any other database bundle.

```xml
<artifactId>org.jkiss.dbeaver.ext.intersystems</artifactId>
  <version>1.0.0-SNAPSHOT</version>
  <packaging>eclipse-plugin</packaging>
</project>
```

and it is all  + all with and before the </parent>

Reply:
I did this in the recent commit